Tools/autotest/quadplane.py

```diff
@@ -2164,6 +2164,90 @@ def WindEstimateConsistency(self):
                 self.progress("Wind estimates correlated")
                 break
 
+    def FullRateClimbout(self):
+        '''test the full rate climbout after VTOL Takeoff functions'''
+        # Set from examining the autotest with enable/disable flag.
+        ALTITUDE_CUTOFF_M = 35
+        CRUISE_ALTITUDE_CUTOFF = 120
+        WAIT_AFTER_CRUISE_DELAY_S = 10
+
+        # Mission planning items.
+        CRUISE_WP_NUM = 2
+
+        self.progress("Testing full rate climbout.")
+
+        def run_test(flag):
+            filename = "mission.txt"
```

Comment:
Note that a lot of newer tests are using `upload_simple_relhome_mission` now-adays.  It's a bit less work and does keep things together.  It has the disavantage of making it harder to load the mission ino SITL (eg. with `sim_vehicle.py`